dyno/panda/dyno_panel.py

```diff
@@ -135,6 +135,7 @@ def push_config(event):
     if READ_SERIAL_ON:
         driver.write(throttle_config)
     print(throttle_config)
+    mcu_flag
```

Comment:
Whats this line supposed to do? It will probably throw an error. Did you mean mcu_flag( )?

Reply:
We dont need a function to do this. Just write `mcu_acknowledged = False` here